src/lib/decodeBySigHash.ts

```diff
@@ -126,6 +99,31 @@ export function parse4BytesResToIfaces(
   return ifaces;
 }
 
+async function safeFetch<T>(...args: Parameters<typeof fetch>): Promise<T> {
+  return fetch(...args).then(async (response) => {
+    if (response.status === 200) {
+      return response.json() as unknown as T;
+    } else {
+      throw new Error(`${response.status} ${response.statusText}`);
+    }
+  });
+}
+
+interface FourBytes {
```

Comment:
Is this the best name?
`FourBytesReponseEntry`, `Signature`, `SignatureDescription`.